EIPS/eip-7685.md

```diff
@@ -45,30 +45,30 @@ Each request type will defines its own `requests` object using with its own
 
 #### Block Header
 
-Extend the header with a new 32 byte value `requests_hash`.
+Extend the header with a new 32 byte commitment value `requests_hash`.
 
-The construction looks like:
+A requests hash list is computed by hashing the elements of the block requests list, but
+only ones where `request_data` is non-empty. Note we assume `request_type` has a size of
+one byte.
 
-```
-sha256(sha256(requests_0) ++ sha256(requests_1) ++ ...)
-```
+The commitment is then computed as the sha256 hash of the list of element hashes.
 
-Or in pseudocode:
 
 ```python
-def compute_requests_hash(requests):
+def compute_requests_hash(block_requests):
```

Comment:
```suggestion
def compute_requests_hash(block_requests: Sequence[bytes]):
```
This is to highlight that `block_requests = [requests_00, requests_01, …]` and not to be messed up with `requests = request_type ++ request_data`